addons/hr_holidays/models/hr_leave.py

```diff
@@ -681,8 +681,9 @@ def _check_date(self):
                                                         date_to=conflicting_holiday_data['date_to'],
                                                         state=conflicting_holiday_data['state'])
                         conflicting_holidays_strings.append(conflicting_holidays_string)
-                    raise ValidationError(_('You can not set two time off that overlap on the same day.\nExisting time off:\n%s') %
-                                          ('\n'.join(conflicting_holidays_strings)))
+                    raise ValidationError(_("You’ve already booked time off which overlaps with this period:" +
+                        "\n".join(conflicting_holidays_strings) +
+                        "Attempting to double-book your time off won't magically make your vacation 2x better!"))
```

Comment:
Your indentation is messy in this file ;)

Reply:
Revert, revert! 👀

Reply:
> well, then we have a good reason to r- it!

Some messages really offend customers' intelligence.

This is an enterprise system, not Discovery kids. Keep it simple